bake/cmd/import.go

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
```

Comment:
current license on project is GPLv3, does this conflict?

Reply:
Yeah, I'll remove it. The generator automatically APL's everything and I forgot it from a few files.